taxi/views.py

```diff
@@ -34,6 +37,26 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer_list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer_list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.edit.DeleteView):
+    model = Manufacturer
+    success_url = reverse_lazy("taxi:manufacturer_list")
+    template_name = "taxi/manufacturer_confirm_delete.html"
```

Comment:
Code Efficiency: Similar to previous comments, you don't need to override `template_name` if it's the same as the default one. By default, Django generates it using the format `<app name>/<model name>_confirm_delete.html`. In this case, it would be `taxi/manufacturer_confirm_delete.html`.